openpype/hosts/max/plugins/publish/validate_frame_range.py

```diff
@@ -36,29 +39,46 @@ def process(self, instance):
         if not self.is_active(instance.data):
             self.log.info("Skipping validation...")
             return
-        context = instance.context
 
-        frame_start = int(context.data.get("frameStart"))
-        frame_end = int(context.data.get("frameEnd"))
-
-        inst_frame_start = int(instance.data.get("frameStart"))
-        inst_frame_end = int(instance.data.get("frameEnd"))
+        frame_range = get_frame_range()
```

Comment:
```suggestion
        frame_range = get_frame_range(asset_doc=instance.data["assetEntity"])
```
This should be using the asset from the instance.
Note that you can publish into different assets so it's not always the current context's instance.

Reply:
I will try it later.